### PR TITLE
Add a command to connect to a TLS server and print its presented certs

### PR DESCRIPTION
**Pull request by mcpherrinm**
r: @christodenny @alokmenghrajani @csstaub

### CODE REVIEW

**Comment by alokmenghrajani** [comment]
👍 

Take a CA as input and dump the reason if verification fails? Maybe also a client cert?

**Comment by mcpherrinm** [comment]
@csstaub also asked for an option to specify a ServerName (say, when you're passing it an IP).

So I'll tack a few more features on.

**Comment by mcpherrinm** [comment]
Actually, CA verification would be nice in *all* codepaths, not just when connecting to a remote server.  I'll make a separate PR for that.

File: main.go [hunk 1]
```diff
@@ -43,6 +44,10 @@ var (
 	dump      = app.Command("dump", "Display information about a certificate.")
 	dumpFiles = dump.Arg("file", "Certificate file to dump (or stdin if not specified).").ExistingFiles()
 	dumpType  = dump.Flag("format", "Format of given input (heuristic guess if not specified).").String()
+
+	connect     = app.Command("connect", "Connect to a server and print its certificate")
+	connectTo   = connect.Arg("server:port", "Hostname or IP to connect to").String()
+	connectName = connect.Flag("name", "Override the server name used for SNI").String()
```
Comment (csstaub) [inline main.go]: Nit: other flag descriptions have periods at end.

**Comment by csstaub** [comment]
LGTM

File: main.go (hunk 1 above)
Comment (mcpherrinm) [inline main.go]: http://www.nytimes.com/2016/06/10/world/europe/period-full-stop-point-whatever-its-called-millennials-arent-using-it.html

**Pull request merged**